### PR TITLE
Port 90_Tower to C.

### PR DESCRIPTION
**Pull request by trwyant**
Actually, though, I started from the Perl port, not only because Perl is more C-like, but because I wrote the Perl port.

### CODE REVIEW

**Comment by coding-horror** [comment]
OK! But C is not memory safe and we are not putting memory unsafe languages in the repo.. you'll need to link to it externally please.

**Comment by coding-horror** [comment]
> but a number of SEGVs later (which is, I think, your point) 

Thank you for your understanding! 🙇 Happy to have a link to it in the readme.md!! And I'll move the other one, if we have any errant C implementations in the repo.

**Comment by coding-horror** [comment]
to be clear, the implementation must be moved outside our repo, because we are only focusing on memory safe languages and languages that are scripting-friendly.

File: 90_Tower/c/tower.c [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+#include <string.h>
+#include <limits.h>
+#include <ctype.h>
+#include <readline/readline.h>
+#include <readline/history.h>
+
+#define MAX_DISKS 7
+
+#ifdef DEBUG
+#define DEBUG_MSG(msg,...) \
```
Comment (zz51) [inline 90_Tower/c/tower.c:22]: 1000000

**Pull request closed without merging**